CentralityiOS/CentralityiOS/ToDoFeedViewController.m

```diff
@@ -0,0 +1,75 @@
+//
+//  ToDoFeedViewController.m
+//  CentralityiOS
+//
+//  Created by Samuel Osa-Agbontaen on 7/11/22.
+//
+
+#import "ToDoFeedViewController.h"
+#import "Parse/Parse.h"
+#import "TaskCell.h"
+#import "TaskObject.h"
+
+@interface ToDoFeedViewController () <UITableViewDelegate, UITableViewDataSource>
+
+@end
+
+@implementation ToDoFeedViewController
+
+- (void)fetchData{
+    // construct query
+    PFQuery *query = [PFQuery queryWithClassName:@"TaskObject"];
+    [query orderByDescending:@"createdAt"];
+    //[query whereKey:@"likesCount" greaterThan:@100];
+    query.limit = 20;
+
+    // fetch data asynchronously
+    [query findObjectsInBackgroundWithBlock:^(NSArray *tasks, NSError *error) {
+        if (tasks != nil) {
+            self.arrayOfTasks = tasks;
+        } else {
+            NSLog(@"%@", error.localizedDescription);
+        }
+        [self.taskTableView reloadData];
+        [self.refreshControl endRefreshing];
+    }];
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section{
+    return self.arrayOfTasks.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    TaskCell *cell = [tableView dequeueReusableCellWithIdentifier:@"TaskCell"];
+    TaskObject *task = self.arrayOfTasks[indexPath.row];
+    cell.task = task;
+    cell.taskNameLabel.text = task.taskTitle;
+    cell.taskDescLabel.text = task.taskDesc;
+    return cell;
+}
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.taskTableView.dataSource = self;
+    self.taskTableView.delegate = self;
+    [self fetchData];
+    
+    self.refreshControl = [[UIRefreshControl alloc] init];
+    [self.refreshControl addTarget:self action:@selector(fetchData) forControlEvents:UIControlEventValueChanged];
+    [self.taskTableView insertSubview:self.refreshControl atIndex:0];
+    
+    // Do any additional setup after loading the view.
+}
+
+/*
```

Comment:
more commented out code to axe!

if some of this stuff is in-progress, you can break it into separate diffs or create separate branches until it's ready.